lib/overcommit/hook/pre_commit/es_lint.rb

```diff
@@ -8,9 +8,22 @@ module Overcommit::Hook::PreCommit
   # somewhere else. Example:
   #
   #   EsLint:
+  #     required_executable: 'npm'
   #     enabled: true
   #     command: ['npm', 'run', 'lint']
   #
+  # And currently this hook supports only eslint compact format, you can configure
+  # this in package.json. Example:
```

Comment:
Sorry, `package.json` isn't a lint script. `package.json` is defined about package and development tool dependency. If you'd like to run `npm run lint`, then user has to define it in `package.json`.

If you'd like to read this, [check this.](https://docs.npmjs.com/misc/scripts)